README.md

```diff
@@ -946,6 +947,32 @@ require 'deimos/test_helpers'
 # Can pass a file path, a string or a hash into this:
 Deimos::TestHelpers.schemas_compatible?(schema1, schema2)
 ```
+### Test Helpers
+
+There are helper methods available to configure Deimos for different types of testing scenarios. 
+Currently there is a helper defined for unit tests and for testing Kafka releated code. You can use it as follows:
+
+```ruby
+# The following can be added to a rpsec file so that each unit 
+# test can have the same settings every time it is run
+around(:each) do |example|
+  Deimos::TestHelpers.unit_test!
+  example.run
+  Deimos.config.reset
+end
+
+
+# Similarly you can use the Kafka test helper
+around(:each) do |example|
+  Deimos::TestHelpers.kafka_test
+  example.run
+  Deimos.config.reset
+end
+```
+
+With the help of these helper methods, rspec examples can be written without having to tinker with Deimos settings.
+This also prevents Deimos setting changes from leaking in to other examples.
```

Comment:
Maybe add an example for how people can configure Deimos manually as well without using these new methods.